cli/src/main/resources/application.yml

```diff
@@ -43,73 +33,106 @@ message:
 
-hono:
-  kafka:
-    commonClientConfig:
-      bootstrap.servers: localhost:9092
-    consumerConfig:
-      group.id: hono-cli
+command:
+  timeoutInSeconds: 60
 
 ---
+# Configuration for the Hono sandbox - see:
+# https://www.eclipse.org/hono/getting-started/
+# Use this profile together with "receiver" or "command"
 
 spring:
   config:
     activate:
-      on-profile: statistic
+      on-profile: sandbox
 
-statistic:
-  interval: 10000
-  autoreset: false
+hono:
+  client:
+    host: hono.eclipseprojects.io
+    port: 15671
+    tlsEnabled: true
+    username: consumer@HONO
+    password: verysecret
+  kafka:
+    commonClientConfig:
+      bootstrap.servers: hono.eclipseprojects.io:9092
+      security.protocol: SASL_PLAINTEXT
+      sasl.mechanism: SCRAM-SHA-256
+      sasl.jaas.config: "org.apache.kafka.common.security.scram.ScramLoginModule required username=\"hono\" password=\"hono-secret\";"
 
 ---
+# Configuration for example deployment in local Kubernetes cluster - see:
+# https://www.eclipse.org/hono/getting-started/
+# Use this profile together with "receiver" or "command"
 
-# For use with the demo certificates.
-# Do not use this profile for certificates that the JVM trusts, e.g. for the Hono sandbox. Simply set hono.client.tlsEnabled 
-# to true. 
 spring:
   config:
     activate:
-      on-profile: ssl
+      on-profile: local
+
 hono:
   client:
-    hostnameVerificationRequired: false
-    trustStorePath: target/config/hono-demo-certs-jar/trusted-certs.pem
-    tlsEnabled: true
+    port: 15672
+    tlsEnabled: false
+    username: consumer@HONO
+    password: verysecret
+  kafka:
+    commonClientConfig:
+      bootstrap.servers: UNCONFIGURED
+      security.protocol: SASL_SSL
+      sasl.jaas.config: "org.apache.kafka.common.security.scram.ScramLoginModule required username=\"hono\" password=\"hono-secret\";"
+      sasl.mechanism: SCRAM-SHA-512
+      ssl.truststore.location: UNCONFIGURED
+      ssl.truststore.password: honotrust
+      ssl.endpoint.identification.algorithm: ""
+
+tenant:
+  id: DEFAULT_TENANT
+device:
+  id: 4711
 
 ---
+# Configuration for Kafka - see:
+# https://www.eclipse.org/hono/getting-started-kafka/
+# Use this profile together with "receiver" or "command"
 
 spring:
   config:
     activate:
-      on-profile: amqp-send,amqp-command
+      on-profile: kafka
 
 hono:
-  client:
-    port: 5672
-
-message:
-  address: telemetry
-  payload: '{"temp": 5}'
+  kafka:
+    commonClientConfig:
+      client.id: hono-cli
+    consumerConfig:
+      group.id: cli
 
 ---
 
 spring:
   config:
     activate:
-      on-profile: command
+      on-profile: statistic
 
-hono:
-  client:
-    port: 15671
+statistic:
+  interval: 10000
+  autoreset: false
 
-command:
-  timeoutInSeconds: 60
+---
 
-tenant:
-  id: DEFAULT_TENANT
+spring:
+  config:
+    activate:
+      on-profile: amqp-send,amqp-command
 
-device:
-  id: 4711
+hono:
+  client:
+    host: hono.eclipseprojects.io
+    port: 5672
```

Comment:
why shouldn't we use the TLS port here? My understanding was that we use secured ports with the sandbox and unsecured ones with the local deployment, or am I mistaken?

Reply:
This Spring profile is for the usage as a southbound client for the AMQP adapter (which should be in a separate app FMPOV). This PR targets the northbound client, so I have not thought much about the southbound client. 
The problem is here again the same as with the northbound client: there are a couple of properties (`host`, `port`, `tlsEnabled`, `hostnameVerificationRequired`) that must then be different between the sandbox and the local deployment and then the documentation needs to address them separately. But you're right and I'll change it and extend the documentation (I don't want to add separate profiles for the environments for the southbound client too).

Reply:
I have found that the situation is more complex:

In the Getting Started Guide, only insecure ports are used. This guide is IMHO particularly important because error sources here easily discourage beginners. Besides, it is the only place in the documentation where the use of the (northbound) CLI is mentioned to some extent.

In the user guide of the AMQP adapter, there is one example with TLS, the one you mentioned below. But in this example, TLS is mandatory, because it should demonstrate the authentication with client certificates. This makes it independent from the other examples in that guide (8x insecure port, 1x secure port). 

Then there is the readme of the CLI module, there also only the insecure port is used.

On the sandbox page is the recommendation to use only the secure ports.

The defaults of the CLI were inconsistent: if you didn't specify a profile at all, the TLS port was selected (but nothing worked, because localhost was set and an invalid truststore was set). In southbound client mode, however, the insecure port has always been the default.

In addition, we had discussed elsewhere to offer only the insecure port on Kafka on the sandbox. 

I wonder if it's not best to use the insecure ports as defaults everywhere. It's not pretty, but it's consistent and changes the previous behavior the least. WDYT?

Reply:
> In the Getting Started Guide, only insecure ports are used. This guide is IMHO particularly important because error sources here easily discourage beginners. Besides, it is the only place in the documentation where the use of the (northbound) CLI is mentioned to some extent.

Then I guess, we should keep the Getting Started guide using the insecure ports.

> I wonder if it's not best to use the insecure ports as defaults everywhere. It's not pretty, but it's consistent and changes the previous behavior the least. WDYT?

Not sure what you are referring to here. Is this about using the insecure port(s) in the CLI's Spring profiles by default?

Reply:
> Not sure what you are referring to here. Is this about using the insecure port(s) in the CLI's Spring profiles by default?

Yes, I mean only the defaults in the CLI's Spring profiles.

Reply:
Then let's do that.